.github/workflows/tests.yml

```diff
@@ -229,7 +229,7 @@ jobs:
           cd bottles
           brew test-bot ${{ needs.setup_tests.outputs.test-bot-formulae-args }}
 
-      - name: Failures summary for brew test-bot --only-formula
+      - name: Failures summary for brew test-bot --only-formulae
```

Comment:
This fixes a typo I introduced in https://github.com/Homebrew/homebrew-core/pull/108365.